deployment/roles/deploy/tasks/deploy-centosmsg.yml

```
@@ -0,0 +1,13 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+---
+- name: Deploy centosmsg
+  k8s:
+    namespace: "{{ project }}"
+    definition: "{{ lookup('template', '../templates/dist2src-centosmsg.yml.j2')  }}"
```

I tried in #82 and the template lookup plugin finds the file in template/

```suggestion
    definition: "{{ lookup('template', 'dist2src-centosmsg.yml.j2')  }}"
```

---

I found an even more elegant way, check the `with_template:` in my [tasks/deploy-worker.yml](https://github.com/packit/dist-git-to-source-git/pull/82/files#diff-6778ef9bd59ff7935ac0ff54081b85bd)